test/lib/dictionaries/api/parameterizer.spec.js

```diff
@@ -4,15 +4,67 @@ var assert = require('assert'),
 var parameterizer = require('../../../../lib/dictionaries/api/parameterizer');
 
 describe('Parameterizer', function() {
-    describe('default', function() {
-        var createMockResponse = function() {
-            return {
-                send: function() {},
-                status: function() {},
-                sendStatus: function() {}
-            }
-        };
+    var createMockResponse = function() {
```

Comment:
I would define that kind of things in a beforeEach. That way you can redefine them if needed for a particular test case without having to worry about the possibility of those changes leaking into other tests.